### ISSUE TITLE
Button inside HyperlinkButton not working

### ISSUE THREAD

**Issue by deepaksm1** [wsick/Fayde#239]
If we are placing Button control inside HyperlinkButton control as a content (see below example). Button command or event does not work in Fayde. Same scenario works in silverlight.

```
<HyperlinkButton TargetName="_blank" NavigateUri="{Binding Path=FileUri}">

   <Button Height="32" Width="32" HorizontalAlignment="Right"
                                                  VerticalAlignment="Top"                                                  
                                                  Tag="Delete"                                                  
                                                  Command="{Binding Path=DeleteFileCommand}"
                                                  CommandParameter="{Binding }" />

  </HyperlinkButton>
```

Using fayde version:0.19.7

**Comment by mikefills** [comment]
Is the Uri given by FIleUri showing in the hyperlink?

**Comment by mikefills** [comment]
Why are you even using a Button inside a HyperlinkButton? 
It doesn't make sense.

**Comment by deepaksm1** [comment]
I am showing file list, After clicking on file should be open in new window and want delete file button inside hyperlink blue box as shown in screenshot.

![filelist](https://cloud.githubusercontent.com/assets/4716238/12814325/3cd4ff1e-cb64-11e5-95b2-23b35d944161.jpg)

**Comment by mikefills** [comment]
I've never seen a UI where a button sits on top of another button.

Anyway, instead of having Button inside HyperlinkButton, why don't you create a Control which wraps the Hyperlink and the delete button?
Something like this
```
<Grid>
     <HyperlinkButton />
      <Button Tag="Delete" VerticalAlignment="Top" HorizontalAlignement="Right"  />
</Grid>
```
Also, note you don't need to put Width and Height on the Button.

**Comment by deepaksm1** [comment]
Yes this is alternate solution, Thanks, but above scenario works in silverlight that's why I was trying it. Anyway I am going to implement your solution. Thanks.
